### DISCUSSION TITLE
vertically rotate sceneview ar model

### DISCUSSION THREAD

**Discussion by S1996ouvik** [sceneview/sceneview#513] [Q&A] [answered]
How to rotate 3d objrct vertically in sceneview ar model

**Comment by ridcore** [comment, marked as answer]
By default you can enable rotation of a modelNode with the isRotationEditable property but I'm not sure it allows you to rotate it vertically.

[ModelNode Class](https://sceneview.github.io/api/sceneview-android/sceneview/io.github.sceneview.node/-model-node/index.html)

[Oficial documentation](https://developers.google.com/ar/design/content/content-manipulation?hl=es-419)

Maybe you can achieve this by modifying the onRotate functions of the modelNode so that the Quaternion is done on the Y axis
